src/app/scripts/cac/control/cac-control-explore.js

```diff
@@ -296,8 +299,6 @@ CAC.Control.Explore = (function (_, $, Geocoder, MapTemplates, HomeTemplates, Ro
         if (exploreLatLng && tabControl.isTabShowing(tabControl.TABS.EXPLORE)) {
             clickedExplore();
         }
-
-        getNearbyPlaces();
```

Comment:
Without this call, times don't load when the Explore view is loaded from URL.  With this call, they get loaded twice.  :frowning: 

Passing `{leading: false}` to the `throttle` above call solves that and seems reasonable to me.